### DISCUSSION TITLE
Feature idea: preferred affixes scoped to item-base

### DISCUSSION THREAD

**Discussion by stefaz** [Lailloken/Exile-UI#169] [Ideas & Feedback]
It would be great if we could save preferred affixes per item-base or item-slot and not globally. Currently the preferred affixes are shown on all item-types even if I would like to have e.g. attack speed only on gloves but not in Mainhand/weapon.

related to this it would also be great if we could save those preferred affixes in profiles so we could have different ones for different characters.

**Comment by Lailloken** [comment]
Yes, I want to implement that at some point. I'll think about profiles, sounds interesting.

**Comment by Lailloken** [comment]
Class-specific highlighting is almost ready to go. I have a question about the suggested profiles: How would you use them? I thought about them a bit but cannot come up with a way how I would personally use them, so I'd like to know more about what needs to be included and how they should work.

**Reply by stefaz** [reply]
I'm thinking about profiles in the context of different character builds and the different stats you would look for on your items, e.g. a DoT build will need other affixes than a Spell/Hit-based build or a crit-build will need other affixes than a poison-build and so on...it would just be nice to load a specific profile when playing a specific build and have the build-specific desired affixed loaded with that profile.

**Reply by Lailloken** [reply]
v1.29.5 now includes class-specific highlighting, and five profile slots that can be switched between. If you have any more ideas or suggestions, feel free to let me know.

**Reply by stefaz** [reply]
this is fantastic! just tested both features and it couldn't be implemented better - huge thanks and keep up the great work! your tool makes leveling/SSF/dealing with rare items so much easier, it's honestly a gamechanger in terms of QoL 👍
